src/request.js

```diff
@@ -225,6 +225,23 @@ class Request {
       this.client.instance().numberInsight.get({level: 'standard', number: number}, this.response.insightStandard.bind(this.response));
     });
   }
+
+  // sending messages
+
+  sendSms(first, second, third, command) {
```

Comment:
Cause it's not `to`, `from` etc. Depending on the number of variables provided it has a different meaning, see line 234 and 236